docs/source/advanced/training_tricks.rst

```diff
@@ -5,16 +5,21 @@
 
 .. _training_tricks:
 
+###############
 Training Tricks
-================
-Lightning implements various tricks to help during training
+###############
+
+Lightning implements various tricks to help during training:
```

Comment:
is there a different name we could use? as a user reading this, i'm nervous that i'm "tricked" by what lightning does, as opposed to learning about tools/techniques lightning offers to make my training smoother

the name was also based around the `TrainerTrainingTricksMixin` which has since been deprecated

Reply:
needs to be something catchy (clickbait).
```
1. Training Tips
2. Effective Training Techniques
3. Effective Training Strategies
4. Training Strategies (maybe not to avoid conflicting with Strategy plugins but not sure)
5. Training Tactics
```
or something better you have in mind?